mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -1,491 +1,1606 @@
+	};
+	class vn_b_air_uh1c_07_03 {
+		tags[] = {"uh1", "transport", "helicopter", "usaf"};
+	};
+	class vn_b_air_uh1c_07_04 {
+		tags[] = {"uh1", "transport", "helicopter", "usmc"};
+	};
+	class vn_b_air_uh1c_07_05 {
+		tags[] = {"uh1", "transport", "helicopter", "frogmen"};
+	};
+	class vn_b_air_uh1c_07_06 {
+		tags[] = {"uh1", "transport", "helicopter", "anzac"};
+	};
+	class vn_b_air_uh1c_07_07 {
+		tags[] = {"uh1", "transport", "helicopter", "tf"};
+	};
+	class vn_b_air_uh1d_02_01 {
+		tags[] = {"uh1","armed","transport","helicopter","army"};
+	};
+	class vn_b_air_uh1d_02_02 {
+		tags[] = {"uh1", "transport", "helicopter", "7th"};
+	};
+	class vn_b_air_uh1d_02_03 {
+		tags[] = {"uh1", "transport", "helicopter", "usaf"};
+	};
+	class vn_b_air_uh1d_02_04 {
+		tags[] = {"uh1","transport","helicopter","usmc"};
+	};
+	class vn_b_air_uh1d_02_05 {
+		tags[] = {"uh1", "transport", "helicopter", "frogmen"};
+	};
+	class vn_b_air_uh1d_02_06 {
+		tags[] = {"uh1", "transport", "helicopter", "anzac"};
+	};
+	class vn_b_air_uh1d_02_07 {
+		tags[] = {"uh1", "transport", "helicopter", "tf"};
+	};
+	class vn_b_air_uh1f_01_03 {
+		tags[] = {"uh1", "transport", "helicopter", "usaf"};
+	};
 
-class wreck_short
-{
-	respawnType = "WRECK";
-	time = 10;
-};
+	//Helo CAS
+	class vn_b_air_ach47_01_01 {
+		tags[] = {"ch47","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_ah1g_01 {
+		tags[] = {"ah1g","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_ah1g_06 {
+		tags[] = {"ah1g","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_oh6a_02 {
+		tags[] = {"oh6a","armed","scout","helicopter","MG"};
+	};
+	class vn_b_air_oh6a_03 {
+		tags[] = {"oh6a","armed","scout","helicopter","GL"};
+	};
+	class vn_b_air_oh6a_05 {
+		tags[] = {"oh6a","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_uh1c_01_01 {
+		tags[] = {"uh1","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_uh1c_02_01 {
+		tags[] = {"uh1","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_uh1c_03_01 {
+		tags[] = {"uh1","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_uh1c_04_01 {
+		tags[] = {"uh1","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_uh1c_05_01 {
+		tags[] = {"uh1","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_uh1c_06_01 {
+		tags[] = {"uh1","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_uh1d_03_06 {
+		tags[] = {"uh1","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_uh1e_01_04 {
+		tags[] = {"uh1","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_uh1e_02_04 {
+		tags[] = {"uh1","armed","firesupport","helicopter"};
+	};
+	class vn_b_air_ch34_04_01 {
+		tags[] = {"ch34","armed","firesupport","helicopter"};
+	};
 
-class wreck_medium
-{
-	respawnType = "WRECK";
-	time = 30;
-};
+	//Plane CAS
+	class vn_b_air_f100d_at {
+		tags[] = {"f100","armed","firesupport","jet"};
+	};
+	class vn_b_air_f4b_at {
+		tags[] = {"f4","armed","firesupport","jet","naval"};
+	};
```

Comment:
this class does not exist -- this is what is causing vehicles to not spawn.

use this to double check vehicle class names -- https://wiki.sogpf.com/index.php/CfgVehicles_WEST

Reply:
resolved